src/utils.js

```diff
@@ -112,3 +114,20 @@ export const requestPromise = (options, iteration = 0) => {
         });
     });
 };
+
+/**
+ * Checks that given parameter is of given type and throws ApifyError.
+ * If errorMessage is not provided then error message is created from name and type of param.
+ *
+ * @param type         string - "String", "Number", ... (see ee: https://github.com/gkz/type-check)
+ * @param value        string - user entered value of that parameter
+ * @param name         string - parameter name (crawlerId for options.crawlerId)
+ * @param errorMessage string - optional error message
+ */
+export const checkParamOrThrow = (type, value, name, errorMessage) => {
```

Comment:
I'd switch the order of `type` and `value` parameters, it easier to follow in the code, especially if the pattern is longer. Also the name `checkParamOrThrow` talks about "param", so the param should have a prominent position.